.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# 📝 Pull Request Title
+
+## 📖 Description
+Please provide a clear and concise description of your changes. Include any relevant background information or context that may help reviewers understand the purpose of this PR.
```

Comment:
Change this to: 
Please provide a general description of your changes. Include any relevant background information or context that may help reviewers understand the purpose of this PR.